domain/consensus/processes/blockvalidator/block_body_in_isolation_test.go

```diff
@@ -21,6 +23,31 @@ import (
 	"github.com/pkg/errors"
 )
 
+func TestBlockValidator_ValidateBodyInIsolation(t *testing.T) {
+	tests := []func(t *testing.T, tc testapi.TestConsensus, cfg *consensus.Config){
+		CheckBlockSanity,
```

Comment:
All of this functions shouldn't be public. And a more proper name will be testCheckBlockSanity IMHO

Reply:
This one has to start with capital T for the go test runner to run it.
they're inside a test so they can't really be "public" anyway, I can rename the *others* to lowercase if you prefer